core/arch/asm_defines.asm

```diff
@@ -328,8 +328,17 @@ ASSUME fs:_DATA @N@\
 #   define LIB_SEG_TLS gs /* keep in sync w/ unix/os_exports.h defines */
 #  endif
 # endif /* 64/32-bit */
+#  define REG_XMM0 xmm0
```

Comment:
There's no other "xmm" so I'm not sure why we need defines like these?  The REG_XAX, etc. are to hide the rax/eax distinction.  There's no variation here.  Wouldn't it be better to just use "xmm0" in the asm?

Reply:
Ok. Renamed to regular names. Previously, REG_ZMM versions had been added. We then might want to remove them as well.